lib/dataSource.js

```diff
@@ -16,138 +16,123 @@ var fluid = require("infusion"),
     kettle = fluid.registerNamespace("kettle"),
     fs = require("fs"),
     http = require("http"),
-    url = require("url");
+    urlModule = require("url"),
```

Comment:
JIRAed at https://issues.fluidproject.org/browse/FLUID-5900 - will add corresponding TODOs. Note remark below that AJV is no kind of alternative to JSONlint.